CONTRIBUTING.md

```diff
@@ -3,8 +3,11 @@
 1. [Fork](https://help.github.com/articles/fork-a-repo/) this repository to your own GitHub account and then [clone](https://help.github.com/articles/cloning-a-repository/) it to your local device.
 2. Install the dependencies: `npm install`.
 3. Run `npm link` to link the local repo to NPM.
-4. Run `nuxt build` to build or `nuxt` to build and watch for code changes.
-5. Then npm link this repo inside any example app with `npm link nuxt`.
-6. Then you can run your example app with the local version of Nuxt.js (You may need to re-run the example app as you change server side code in the Nuxt.js repository).
+4. `nuxt build` to build and `nuxt` to build and watch for code changes are now globally available but only works inside a nuxt project.
```

Comment:
We don't need `nuxt build` anymore. (At lease currently)

Reply:
You mean that the correct way is to run `npm run build` or `npm run dev` in the  project that is using the linked nuxt folder? In that case I suggest that we just remove the line.

Reply:
Oh no sorry. I was too tired when reviewing this.